client/src/Dataset/List/ListCard.tsx

```diff
@@ -32,7 +36,7 @@ const DatasetCard: React.FC<Props> = ({ item, onClick, renderMenuItems }) => {
                     className={classes.image}
                     image={
                         item.first_image_id != null
-                            ? `/api/image/${item.first_image_id}/"?width=250"`
+                            ? `/api/image/${item.first_image_id}?width=250`
```

Comment:
it's better to encapsulate it in the function that returns image url, so that the component doesn't know where the image itself lives and it can be easily changed in the future. Create a function that will get `first_image_id` as a param